Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
```

Comment:
You can use a pattern for these two files to avoid the extra layer. Also, since the `WORKDIR` is already set to /usr/src/app above, you can use ./ for the second argument:

```dockerfile
COPY composer.* ./
```